src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -612,6 +612,13 @@ public ReportNode modifyInjectionConnectivityAttributes(ConnectablePosition<?> c
                                                       InjectionModificationInfos modificationInfos,
                                                       ReportNode connectivityReports) {
         List<ReportNode> reports = new ArrayList<>();
+        if (modificationInfos.getVoltageLevelId() == null || modificationInfos.getBusOrBusbarSectionId() == null) {
+            return ReportNode.newRootReportNode()
+                    .withMessageTemplate("VoltageLevelOrBusNotFound", "Voltage Level or Bus with id=${id} not found")
```

Comment:
```suggestion
        .withMessageTemplate("VoltageLevelOrBusbarSectionNotFound", "Voltage level id or Bus bar section id of Equipment with id=${id} not found")
```
As I remember, with BUS_BREAKER topology, we have a future ticket Jira to remove connectivity form, so it is better with Bus bar section in the message